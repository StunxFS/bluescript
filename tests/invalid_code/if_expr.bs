fn main() {
    if 1 { }
}