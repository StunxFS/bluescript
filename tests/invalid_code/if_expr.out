tests/invalid_code/if_expr.bs:2:8: error: non-boolean `if` condition
